### PR TITLE
Remove IS_LOW_MEM around comment_object::parent #799

### PR DESCRIPTION
**Pull request by maslenitsa93**

### CODE REVIEW

File: libraries/chain/include/golos/chain/comment_object.hpp [hunk 1]
```diff
@@ -240,7 +240,6 @@ namespace golos {
                         member<comment_object, comment_id_type, &comment_object::id>>,
             composite_key_compare <std::less<account_name_type>, strcmp_less, std::less<comment_id_type>> >
         /// NON_CONSENSUS INDICIES - used by APIs
-#ifndef IS_LOW_MEM
```
Comment (afalaleev) [inline libraries/chain/include/golos/chain/comment_object.hpp]: Why do you remove these lines, which disable non-consensus indicies?

File: plugins/social_network/social_network.cpp [hunk 2]
```diff
@@ -253,7 +253,7 @@ namespace golos { namespace plugins { namespace social_network {
         uint32_t vote_limit
     ) const {
         std::vector<discussion> result;
-#ifndef IS_LOW_MEM
```
Comment (afalaleev) [inline plugins/social_network/social_network.cpp]: Why do you remove this `ifndef IS_LOW_MEM`?

Comment (maslenitsa93) [reply plugins/social_network/social_network.cpp]: @afalaleev Why do not remove it? Does it stores any memory, or any another cause to don't include it to LOW_MEM (when it depends on parent_author, and we enabled it)?

**Comment by afalaleev** [comment]
Please rebase your changes to one commit.

**Pull request merged**